packages/@dcl/sdk-commands/src/commands/quests/index.ts

```diff
@@ -0,0 +1,279 @@
+    if (!new RegExp(urlRegex).test(options.args['--target']) && !options.args['--target'].includes('localhost')) {
+      throw new CliError('The provided target is not a valid URL')
+    }
+  }
+  const baseURL = options.args['--target'] || 'https://quests.decentraland.org'
+
+  logger.info(`You're using ${baseURL} server`)
+
+  if (options.args['--create']) {
+    await executeCreateSubcommand(options.components, linkerOpts, baseURL)
+  } else if (options.args['--create-from-json'] && options.args['--create-from-json'].length) {
+    await executeCreateSubcommand(options.components, linkerOpts, baseURL, options.args[`--create-from-json`])
+  } else if (options.args['--list']?.length) {
+    await executeListSubcommand(options.components, linkerOpts, baseURL, options.args['--list'])
+  } else if (options.args['--activate']?.length) {
+    await executeActivateSubcommand(options.components, linkerOpts, baseURL, options.args['--activate'])
+  } else if (options.args['--deactivate']?.length) {
+    await executeDeactivateSubcommand(options.components, linkerOpts, baseURL, options.args['--deactivate'])
+  }
+}
+
+async function executeCreateSubcommand(
+  components: CliComponents,
+  linkerOpts: Omit<LinkerdAppOptions, 'uri'>,
+  baseURL: string,
+  path?: string
+) {
+  const { logger, fs } = components
+  let quest: CreateQuest | null = null
+  if (path) {
+    if (await fs.fileExists(path)) {
+      const createQuestJson = await fs.readFile(path, { encoding: 'utf-8' })
+      try {
+        quest = JSON.parse(createQuestJson) as CreateQuest
+      } catch (error) {
+        throw new CliError(`${path} doesn't contain a valid JSON`)
+      }
+
+      if (!validateCreateQuest(quest, { logger })) {
+        throw new CliError('You provided an invalid Quest JSON. Please check the documentation')
+      }
+    } else {
+      throw new CliError("File doesn't exist")
+    }
+  } else {
+    quest = await createQuestByPrompting({ logger })
+    if (!quest) {
+      throw new CliError('Quest creation was cancelled')
+    }
+  }
+
+  const createURL = `${baseURL}/api/quests`
+
+  await executeSubcommand(
+    components,
+    linkerOpts,
+    {
+      url: createURL,
+      method: 'POST',
+      metadata: quest,
+      actionType: 'create',
+      extraData: { questName: quest.name, createQuest: quest }
+    },
+    async (authchainHeaders) => {
+      try {
+        const res = await fetch(createURL, {
+          method: 'POST',
+          headers: {
+            ...authchainHeaders,
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify(quest)
+        })
+        const questsId: { id: string } = await res.json()
+        logger.log(' ')
+        logger.log(
+          `${colors.greenBright(`> Your Quest: ${quest!.name} was created successfully - ID:`)} ${questsId.id}`
+        )
+        logger.log(' ')
+      } catch (error) {
+        logger.error('> Error returned by Quests Server: ', error as any)
```

Comment:
https://github.com/decentraland/js-sdk-toolchain/blob/c9304695f5e7657c800a0477ce064c0ee4334e99/packages/%40dcl/sdk-commands/src/logic/beautiful-logs.ts#L54

```suggestion
        printError(logger, 'Error returned by Quests Server', error)
```